apps/backend/src/evaluations/evaluations.controller.ts

```diff
@@ -78,23 +78,26 @@ export class EvaluationsController {
   async create(
     @Body() createEvaluationDto: CreateEvaluationDto,
     @UploadedFile() data: Express.Multer.File,
-    @Request() request: {user: User}
+    @Request() request: {user: User; headers: {origin: string}}
```

Comment:
This requires the Origin header to be set by anything interacting with the API to set an Origin header if the caller wants a non-relative shareURL (and if another web-app interacts with Heimdall's API this might set the share URL to the wrong host)